libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.m

```diff
@@ -89,7 +89,9 @@ - (void)viewDidLoad {
 
 - (void)viewDidLayoutSubviews {
     [super viewDidLayoutSubviews];
-    [self layoutViews];
+    if (self.navigationController == nil) {
```

Comment:
Is it possible to do some refactor and consolidate this check to a singular place... Seems scattered all over the place...